apps/web/pages/router/index.tsx

```diff
@@ -8,7 +8,7 @@ import PageWrapper from "@components/PageWrapper";
 
 import { getServerSideProps } from "../../server/lib/router/getServerSideProps";
 
-export default function Router({ form, message }: inferSSRProps<typeof getServerSideProps>) {
+export default function Router({ form, message, errorMessage }: inferSSRProps<typeof getServerSideProps>) {
```

Comment:
Now `message` and `errorMessage` are differentiated, earlier message itself had `errorMessage`